File: data/data.code.typescript.client/AckSceneEnter.ts
namespace proto {
export class AckSceneEnter
{
	private _player: MsgScenePlayer;


	constructor(packet: net.Packet) {
		this._player = new MsgScenePlayer(packet);
	}


	public get player(): MsgScenePlayer { return this._player; }
	public set player(value: MsgScenePlayer) { this._player = value; }
}
}

File: examples/golang/client/proto/ack_scene_enter.go
package proto

import (
	"packet"
)

type AckSceneEnter struct {
	Player                   *MsgScenePlayer
}

func AckSceneEnterDecode(pack *packet.Packet) *AckSceneEnter {
	ackSceneEnter := &AckSceneEnter{}

	ackSceneEnter.Player = MsgScenePlayerDecode(pack)
	return ackSceneEnter
}
